### PR TITLE
Add global shops/read role

### PR DESCRIPTION
**Pull request by loan-laux**
Add a new "read all shops" role for [api-plugin-shops#6](https://github.com/reactioncommerce/api-plugin-shops/pull/6).

### CODE REVIEW

File: migrations/5.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    slug: { $in: affectedGlobalGroups }
+  }, {
+    $addToSet: { permissions: { $each: newGlobalPermissions } }
+  });
+
+  progress(100);
+}
+
+export default {
+  down: "impossible",
```
Comment (focusaurus) [inline migrations/5.js]: Could we use `$pull` to implement down migration?

Comment (loan-laux) [reply migrations/5.js]: Done @focusaurus. Let me know if something else is needed!

**Comment by loan-laux** [comment]
@kieckhafer @rosshadden, since you're taking over community work this week, feel free to give this a final review.

**Review by kieckhafer — APPROVED** [review]
👍 code looks good, works as expected.

**Pull request merged**